lib/chat_web/live/main_live/page/admin_panel.ex

```diff
@@ -166,7 +175,7 @@ defmodule ChatWeb.MainLive.Page.AdminPanel do
       |> Enum.map(&{&1.hash, &1})
       |> Map.new()
 
-    full_user_list = User.list()
+    full_user_list = UsersBroker.list()
```

Comment:
may we incorporate these brokers into `Users` and `Rooms` contexts?

Reply:
sure